src/jupyter_contrib_nbextensions/nbextensions/runtools/cellstate.js

```diff
@@ -0,0 +1,20 @@
+
+
+  CodeMirror.defineOption("cellstate", false, function(cm, val, old) {
+    if (old && old != CodeMirror.Init) {
+      cm.clearGutter(cm.state.cellState.options.gutter);
+      cm.state.cellState = null;
+      cm.off("gutterClick", onGutterClick);
+      cm.off("change", onChange);
+      cm.off("viewportChange", onViewportChange);
+      cm.off("swapDoc", onChange);
+    }
+    if (val) {
+      cm.state.cellState = new State(parseOptions(val));
+      updateInViewport(cm);
+      cm.on("gutterClick", onGutterClick);
```

Comment:
are these defined in this scope? How does this file get loaded?

Reply:
It is loaded at `load_extension` time now.